SYCL/XPTI/Inputs/test_collector.cpp

```diff
@@ -6,21 +6,52 @@
 
 std::mutex GMutex;
 
+XPTI_CALLBACK_API void syclBufferCallback(uint16_t, xpti::trace_event_data_t *,
+                                          xpti::trace_event_data_t *, uint64_t,
+                                          const void *);
```

Comment:
Maybe it's better to move this code to https://github.com/intel/llvm-test-suite/blob/intel/SYCL/XPTI/Inputs/buffer_info_collector.cpp and test both high-level and low-level mappings together?